src/System.Net.Http/src/System/Net/Http/Managed/HttpConnection.cs

```diff
@@ -308,7 +308,7 @@ private Task WriteFormattedInt32Async(int value, CancellationToken cancellationT
                 }
 
                 // Parse the response status line.
-                var response = new HttpResponseMessage() { RequestMessage = request, Content = new HttpConnectionContent(CancellationToken.None) };
+                var response = new HttpResponseMessage() { RequestMessage = request, Content = new HttpConnectionContent(cancellationToken) };
```

Comment:
Changed to `cancellationToken` in place of `CancellationToken.None` because the token passed to `ReadNextLineAsync` is not further passed down into its call chain and hence there is no way to dispose the underlying streams to trip the operations via an exception.